sklearn/tests/test_docstring_parameters.py

```diff
@@ -20,10 +20,18 @@
     enable_iterative_imputer,  # noqa
 )
 from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import (
+    f1_score,
+    fbeta_score,
+    precision_recall_fscore_support,
+    precision_score,
+    recall_score,
+)
```

Comment:
It might be better to change the import here and have

```python
from sklearn import metrics
```

and then call `metrics.f1_score`.

We will end-up importing all functions of scikit-learn maybe :)

Reply:
Maybe we have to see if we want to isolate all those consistency checks in the future if there are too many.

Reply:
Yes, I originally put these with the metrics tests (see: https://github.com/scikit-learn/scikit-learn/pull/28678#discussion_r1715025485), as I had the same thought - potentially we will check many docstrings, and it may make more sense to put them with their own tests. The other tests in `test_docstring_parameters.py` are more general, and covers most of the public classes/functions.

But I am not familiar with a lot of the codebase, so I do not know how many more places we would want to use this test. We can always move later!